full-stack-tests/rpc-interface/webpack.config.js

```diff
@@ -42,6 +42,8 @@ module.exports = {
   },
   externals: {
     electron: "commonjs electron",
+    stream: {},
+    zlib: {},
```

Comment:
Will this be necessary in all frontends?  Is display-test-app still able to build and run with these changes?

Reply:
`display-test-app` and `presentation-test-app` both work without modifications because they are built with `react-scripts` which automatically polyfills NodeJs apis. Starting Webpack 5, this is no longer the case, so I'll check if the code still compiles on our cra5 branch.

Reply:
Removed the offending top-level imports, they are now `require`d at runtime only in backend.